doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+### Use case: Rename tag of person
+
+**MSS**
+
+1. User requests to rename tag of a specific person
+2. AddressBook shows confirmation prompt
+3. User confirms request to rename tag
+4. AddressBook renames the existing tag with a new tag
+Use case ends.
+
+**Extensions**
+
+2a. No person with the tag
+
+> 2a1. AddressBook shows error message
+> Use case ends
+
+4a. User cancels request to rename tag
```

Comment:
Here, it should be `2a.` because it stops you from going to `3. User confirms request to rename tag`